QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.Web/wwwroot/js/home.js

```
@@ -1,41 +1,102 @@
+                $("#fareResult").show();
+
+                // Reset button state
+                submitBtn.html(originalButtonText);
+                submitBtn.prop('disabled', false);
+                submitBtn.removeClass('disabled');
+            },
+            error: function (xhr, status, error) {
+                // Handle error
+                let errorMessage = 'An error occurred while calculating the fare';
+
+                if (xhr.responseJSON && xhr.responseJSON.message) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)